packages/react-server-dom-webpack/src/ReactFlightServerWebpackBundlerConfig.js

```diff
@@ -18,7 +18,7 @@ export type BundlerConfig = WebpackMap;
 export type ServerReference<T: Function> = T & {
   $$typeof: symbol,
   $$id: string,
-  $$bound: Array<ReactModel>,
+  $$bound: Array<ReactClientValue>,
```

Comment:
why isn't the type `$$bound: null | Array<ReactClientValue>`?

in `getServerReferenceBoundArguments` it is passed this type and reads this value but then after calling that you null check the return value. I believe it can return null and that this type is wrong but I'm not sure if I'm just missing something?

Reply:
Yea, it's an oversight. It gets assigned null in the register/loader generated code that isn't type checked.